core/src/main/scala-3/cats/derived/package.scala

```diff
@@ -0,0 +1,44 @@
+package cats.derived
+
+import cats.*
+
+object semiauto extends
+  CommutativeMonoidDerivation,
+  CommutativeSemigroupDerivation,
+  ContravariantDerivation,
+  EmptyDerivation,
+  EmptyKDerivation,
+  EqDerivation,
+  FoldableDerivation,
+  FunctorDerivation,
+  HashDerivation,
+  InvariantDerivation,
+  MonoidDerivation,
+  MonoidKDerivation,
+  OrderDerivation,
+  PartialOrderDerivation,
+  ReducibleDerivation,
+  SemigroupDerivation,
+  SemigroupKDerivation,
+  ShowDerivation,
+  TraverseDerivation,
+  Instances:
+
+  inline def foldable[F[_]](using ev: DerivedFoldable[F]): Foldable[F] = ev.instance
```

Comment:
Sorry, this is what I meant:
```suggestion
  inline def foldable[F[_]]: Foldable[F] = DerivedFoldable[F]
```